hipogram/posts/models.py

```diff
@@ -32,6 +32,9 @@ class Like(models.Model):
     post = models.ForeignKey("Post", on_delete=models.CASCADE, related_name="likes")
     user = models.ForeignKey("auth.User", on_delete=models.CASCADE)
 
+    def __str__(self):
+        return self.user.username
```

Comment:
"Seyma liked Admin's post" would be better IMO. Also "Seyma rated 3 to Admin's post" for Rate.